theme_tutorial/views/pages.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
![image](https://github.com/AntonioMartinD/Building_a_website/assets/35231827/ecfef47d-23ad-4f97-a443-511101f99013)
This is the structure for the pages, templates and views